### PR TITLE
Feature/load outfit

### PR DESCRIPTION
**Pull request by ERaith**
Created new function called loadSavedOutfit. 
+ is called from removeCard( Should change this name or add a helper function in future to be more semantic.) when the savedOutfit card is clicked on.
+ Takes the id from the nearest parent that matches the criteria and calls showGarmentOnBear when not null
+ sets the currentOutfit to the saved outfit
+ updates the box name.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -135,5 +135,22 @@ function removeCard(event) {
...
+    var outfitId = event.target.closest('.saved_outfit').id;
+    loadSavedOutfit(outfitId);
+  }
+}
+
+function loadSavedOutfit(id){
+  clearBearDisplay();
+  for(var i =0; i<outfits.length; i++){
+    if(outfits[i].id==id){
+      currentOutfit = outfits[i];
```
Comment (jonathan-tschida) [inline main.js]: If you save an outfit, reload the page, then load the outfit, you can no longer edit the current outfit, I think this line right here might be causing the problem

Comment (jonathan-tschida) [reply main.js]: I'm getting an error saying that currentOutfit.addGarment isn't a function, so it seems like you're overwriting currentOutfit somehow

Comment (ERaith) [reply main.js]: Fixed that. Good catch.

Comment (ERaith) [reply main.js]: used Object.assign to assign all values from one saved object to the current object.

**Comment by jonathan-tschida** [comment]
Great work guys! We did it!

**Pull request merged**